src/PrefixingContainer.php

```diff
@@ -58,8 +59,12 @@ public function __construct(ContainerInterface $container, string $prefix, bool
      */
     public function get($key)
     {
+        if (!$this->isPrefixed($key) && $this->strict) {
+            throw new NotFoundException("Key '{$key}' does not exist", 0, null, $this, $key);
```

Comment:
Not a big deal, but I'd prefer `sprintf()` instead of double quotes, because it is more easily internationalized when necessary.